src/utils.ts

```diff
@@ -123,8 +123,9 @@ function getStartDir(): string {
 
   const grandparent = path.resolve(__dirname, '..', '..');
   if (grandparent.endsWith('/node_modules')) {
-    return path.resolve(grandparent, '..');
+    return process.cwd();
```

Comment:
If we're using `process.cwd()`, there's no need for the `getStartDir` function at all. `process.cwd()` is synonymous with `.` so the conditions in the function don't make any sense.

It would be  better to remove this function and replace all calls to `getStartDir` with `process.cwd()`. Alternatively, just replace the contents of  `getStartDir` with 

```
return process.cwd()
```

Reply:
it seems like that to me but,
When i change the function to just `return process.cwd()`
more test fails

I assume it because absolute and relative paths are handled differently in some point in the code. 
If its important i can try to figure what's going on.

Reply:
You're right. This change is good in its current form.